po/ja.po

```diff
@@ -16621,18 +16623,26 @@ msgid ""
 "channel for the new message. The thread can be blocked indefinitely if there "
 "is nobody who reads from the channel."
 msgstr ""
+"`send`を呼ぶことによって、チャネルに新たなメッセージのための空きができるま"
+"で、現在のスレッドはブロックされます。もし誰もチャネルから値を読み取らない場"
+"合は、このスレッドは無期限にブロックされることがあります。"
```

Comment:
I would say something like: "Sendを呼んだときにチャネルに空きがなければ、チャネルに..."
although that will not be a 1-to-1 translation

Reply:
The English original can be improved in many cases — as translators, you're in a unique position to do so since you'll be reading the text more carefully than most :smile: 

You can always fix small issues directly in the translation. It would then be helpful to "port" those fixes back to the English text. When you `msgmerge` the PO file next, you will see that the translation is marked "fuzzy". This signals that it needs proof-reading: in this case there will be nothing to do except remove the fuzzy marker again. This kind of de-fuzzing should be done repeatedly, ideally weekly or so.

Reply:
Oh, second small point: GitHub has a great suggestion tool which saves a lot of time in discussions. It's the left-most button in the toolbar and it looks like this when I add what you wrote above (no idea what it says):

```suggestion
"Sendを呼んだときにチャネルに空きがなければ、チャネルに"
```

People can commit suggestions with a few clicks, which saves a lot of time.